Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 560, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------

window = BayesRegression(df, y='isi', treatment='stim', condition=['stim_strength','neuron_x_mouse'],
                              group='mouse')
window.fit(model=models.model_hierarchical)

------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
[0;32m/tmp/ipykernel_8070/2649984714.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m window = BayesRegression(df, y='isi', treatment='stim', condition=['stim_strength','neuron_x_mouse'],
[1;32m      2[0m                               group='mouse')
[0;32m----> 3[0;31m [0mwindow[0m[0;34m.[0m[0mfit[0m[0;34m([0m[0mmodel[0m[0;34m=[0m[0mmodels[0m[0;34m.[0m[0mmodel_hierarchical[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/bayes_window/slopes.py[0m in [0;36mfit[0;34m(self, model, do_make_change, fold_change_index_cols, do_mean_over_trials, fit_method, add_condition_slope, **kwargs)[0m
[1;32m     53[0m         [0mself[0m[0;34m.[0m[0mmodel[0m [0;34m=[0m [0mmodel[0m[0;34m[0m[0;34m[0m[0m
[1;32m     54[0m         [0;32mif[0m [0mfold_change_index_cols[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 55[0;31m             [0mfold_change_index_cols[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0mwindow[0m[0;34m.[0m[0mlevels[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     56[0m         [0mfold_change_index_cols[0m [0;34m=[0m [0mlist[0m[0;34m([0m[0mfold_change_index_cols[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     57[0m         if self.window.detail and (self.window.detail in self.window.data.columns) and (

[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pandas/core/generic.py[0m in [0;36m__getattr__[0;34m(self, name)[0m
[1;32m   5485[0m         ):
[1;32m   5486[0m             [0;32mreturn[0m [0mself[0m[0;34m[[0m[0mname[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 5487[0;31m         [0;32mreturn[0m [0mobject[0m[0;34m.[0m[0m__getattribute__[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mname[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   5488[0m [0;34m[0m[0m
[1;32m   5489[0m     [0;32mdef[0m [0m__setattr__[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mname[0m[0;34m:[0m [0mstr[0m[0;34m,[0m [0mvalue[0m[0;34m)[0m [0;34m->[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;31mAttributeError[0m: 'DataFrame' object has no attribute 'levels'
AttributeError: 'DataFrame' object has no attribute 'levels'

